packages/paste-website/src/components/shortcodes/component-header/index.tsx

```diff
@@ -3,7 +3,12 @@ import {Anchor} from '@twilio-paste/anchor';
 import {Box} from '@twilio-paste/box';
 import {Text} from '@twilio-paste/text';
 import {Heading} from '@twilio-paste/heading';
+import {Stack} from '@twilio-paste/stack';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
 import {Breadcrumb, BreadcrumbItem} from '../../breadcrumb';
```

Comment:
Note to self, use our actual breadcrumb component

Reply:
I've added a ticket to the backlog. If helper duty is light next week, I'll knock it out.